.github/workflows/inferno-test.yml

```diff
@@ -0,0 +1,74 @@
+name: Inferno Certification Test
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+        branches: [ master ]
+
+permissions:
+    contents: read
+
+jobs:
+    inferno_certification_test:
+        runs-on: ubuntu-22.04
+        name: Inferno Certification Test
+        timeout-minutes: 180  # 3 hours timeout due to potential terminology downloads
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  submodules: 'recursive'
+                  token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
note PRs have no access to secrets since github essentially sees it as outside openemr's repo.